doc/examples/rendering/perLocationShaderVariationArnold.gfr

```diff
@@ -442,35 +441,52 @@ Gaffer.Metadata.registerValue( __children["Scene"]["BoxOut"]["__out"], 'nodule:t
-__children["MixColor"]["parameters"]["a"]["g"].setInput( __children["user_data_rgb_readColour"]["out"]["g"] )
-__children["MixColor"]["parameters"]["a"]["b"].setInput( __children["user_data_rgb_readColour"]["out"]["b"] )
+__children["MixColor"]["parameters"]["a"].setInput( __children["UserDataRgb"]["out"] )
+__children["MixColor"]["parameters"]["a"]["r"].setInput( __children["UserDataRgb"]["out"]["r"] )
+__children["MixColor"]["parameters"]["a"]["g"].setInput( __children["UserDataRgb"]["out"]["g"] )
+__children["MixColor"]["parameters"]["a"]["b"].setInput( __children["UserDataRgb"]["out"]["b"] )
 __children["MixColor"]["parameters"]["b"].setValue( imath.Color3f( 0.819999993, 0.819999993, 0.819999993 ) )
 __children["MixColor"]["parameters"]["m"].setInput( __children["mx_smoothstep_float"]["out"]["out"] )
 __children["MixColor"]["__uiPosition"].setValue( imath.V2f( -83.8771439, -38.4639282 ) )
 __children["Scene"]["Cow_Field"]["Expression"]["__engine"].setValue( 'python' )
 __children["Scene"]["Cow_Field"]["Expression"]["__expression"].setValue( 'import imath\n\nparent["__out"]["p0"] = parent["__in"]["p0"]' )
+__children["Backdrop6"] = Gaffer.Backdrop( "Backdrop6" )
+parent.addChild( __children["Backdrop6"] )
+__children["Backdrop6"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Backdrop6"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+Gaffer.Metadata.registerValue( __children["Backdrop6"], 'nodeGadget:color', imath.Color3f( 0.347949386, 0.438627928, 0.5 ) )
+__children["Backdrop6"]["title"].setValue( 'Note: Randomizing Color3f plugs' )
+__children["Backdrop6"]["description"].setValue( "When randomizing Color3f plugs, make sure to right-click the color preview to the right of the channel fields to target the whole plug. If you right-click one of the channel fields, you'll only randomize that one channel." )
+__children["Backdrop6"]["__uiBound"].setValue( imath.Box2f( imath.V2f( 0, 0 ), imath.V2f( 40, 20 ) ))
+__children["Backdrop6"]["__uiPosition"].setValue( imath.V2f( -25, 27.75 ) )
+__children["Backdrop7"] = Gaffer.Backdrop( "Backdrop7" )
+parent.addChild( __children["Backdrop7"] )
+__children["Backdrop7"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Backdrop7"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+Gaffer.Metadata.registerValue( __children["Backdrop7"], 'nodeGadget:color', imath.Color3f( 0.347949386, 0.438627928, 0.5 ) )
+__children["Backdrop7"]["title"].setValue( 'Note: Renderers and custom attributes' )
```

Comment:
Is the title meant to be wider than the backdrop:

![image](https://user-images.githubusercontent.com/896779/62478477-187e6680-b7a3-11e9-9836-b135adaf277f.png)

Reply:
Not many options, here. In the examples I've made, I've kept the backdrop body text constrained to ~80 chars per line for readability. Adding an extra wide title-only backdrop behind the main backdrop leaves lots of empty space between the title and the text... not sure how else we could address this.

Reply:
Scratch that. Fixed in dcc75916794581692cf2acf81feee10954c833ec.